static/app/views/conduit/conduitDemo.stories.tsx

```diff
@@ -0,0 +1,71 @@
+import {useMemo, useState} from 'react';
+import {useStream} from 'conduit-client';
+
+import {Heading} from '@sentry/scraps/text';
+
+import {Button} from 'sentry/components/core/button';
+import {Flex} from 'sentry/components/core/layout/flex';
+import {Text} from 'sentry/components/core/text/text';
+import * as Storybook from 'sentry/stories';
+import getCsrfToken from 'sentry/utils/getCsrfToken';
+import useOrganization from 'sentry/utils/useOrganization';
+
+type Message = {
+  value: string;
+};
+
+export default Storybook.story('Conduit Demo', story => {
+  story('Streaming Demo', () => {
+    const organization = useOrganization();
+    const [messages, setMessages] = useState<Message[]>([]);
+    const [isEnabled, setIsEnabled] = useState(false);
+
+    const streamUrl = useMemo(
+      () => `/api/0/organizations/${organization.slug}/conduit-demo/`,
+      [organization.slug]
+    );
```

Comment:
```suggestion
    const streamUrl = `/api/0/organizations/${organization.slug}/conduit-demo/`;
```
You can discard the memo here, this is a primitive that will diff just fine if inlined